i18n/en.ts

```diff
@@ -215,7 +215,7 @@ const text: Translations = {
   Section: {
     about: 'ABOUT',
     gallery: 'GALLERY',
-
+    title: 'Sections',
```

Comment:
Make a new attribute `Sections` and put `title` in that. `Section` is reserved for the dedicated section page(s) we have.